### ISSUE TITLE
Can the web be independent

### ISSUE THREAD

**Issue by baihewujiecao** [trexminer/T-Rex#1252]
Can the web be independent

**Comment by dedizones** [comment]
yes via slave connections (daemon miner) of the miner on a centralized web (master web) such as monitoring solutions

**Comment by Conzar** [comment]
What is the configuration for setting up the daemon miner?

**Comment by dedizones** [comment]
Configuration basic hiveos (no setting, blank) in and last update miner

OC setting 
Core 1490
Mem 2400
Pl 0

this increase in watt consumption is really when the block is generated then it returns to normal

I see it on my connected socket which displays the results in real time

**Comment by Conzar** [comment]
> daemon miner

Sorry, I do not understand.
I was asking for the configuration to start T-Rex without any mining configured and only running the web ui.  Does the configuration you listed still require a GPU and mining?
